1001-2000/LIT1982Mashaf.xml

```diff
@@ -4,7 +4,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">Mäṣḥafä ḍäḥay</title>
+            <title xml:id="t1">Maṣḥafa ̣Ṣ́aḥāy</title>
```

Comment:
title in gez
title normalized
title translated

Reply:
check https://betamasaheft.eu/Guidelines/?id=transliteration-principles 
ḍ is correct

Reply:
A encoded the Gəʿəz-title with xml:lang="gez"; the transliteration I encoded as type="normalized" (but not: type="transliterated", what was also offered by Oxygen); The English title I encoded with xml:lang="en"
```
<titleStmt>
            <title xml:lang="gez" xml:id="t1">መጽሐፈ፡ ፀሓይ፡ </title>
            <title type="normalized" xml:id="t2">Maṣḥafa ḍaḥāy</title>
            <title xml:lang="en" xml:id="t3">Book of the Sun</title>
            <editor role="generalEditor" key="AB"/>
            <funder>Akademie der Wissenschaften in Hamburg</funder>
         </titleStmt>
```
Is this all correct?